types.d.ts

```diff
@@ -245,3 +254,17 @@ declare type EventsJournalPollingOptions = {
     interval?: number;
 };
 
+/**
+ * @typedef {object} SignatureOptions
+ * @property {string} [digiSignature1] Value of digital signature retrieved from the x-adobe-digital-signature1 header in each POST request to webhook
+ * @property {string} [digiSignature2] Value of digital signature retrieved from the x-adobe-digital-signature2 header in each POST request to webhook
+ * @property {string} [publicKeyUrl1] Value of public key url retrieved from the x-adobe-public-key1-url header in each POST request to webhook
+ * @property {string} [publicKeyUrl2] Value of public key url retrieved from the x-adobe-public-key2-url header in each POST request to webhook
+ */
+declare type SignatureOptions = {
+    digiSignature1?: string;
+    digiSignature2?: string;
+    publicKeyUrl1?: string;
+    publicKeyUrl2?: string;
```

Comment:
`?` signifies that the parameter mentioned is optional. Since that is not the case here, can you please remove them.